app/views/admin/sdg/managers/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<h2><%= t("admin.sdg_managers.index.title") %></h2>
+
+<%= render "admin/shared/user_search", url: search_admin_sdg_managers_path %>
```

Comment:
Since we're at it, how about including the searched term in the search input form? The fact that it stays blank after a search puzzles me every time :sweat_smile:.